pyvista/plotting/camera.py

```diff
@@ -31,6 +31,38 @@ def __init__(self):
         self._elevation = 0.0
         self._azimuth = 0.0
 
+    def __repr__(self):
+       """Print a repr specifying the id of the camera and its camera type."""
+       return (f'<{self.__class__.__name__} at {hex(id(self))}>')
```

Comment:
I suggest using as few dunders as possible. But is this necessary here? Is this not the same as the default `repr`? On a related note: what is "camera type" in the docstring?
```suggestion
       return (f'<{type(self).__name__} at {hex(id(self))}>')
```

Reply:
If we're including a __repr__, we could always include information regarding the camera.  I'm not opposed to that considering how it's nice to be able to get a "report" of the state of an instance.

Reply:
Yeah, that would work too. We could even switch to a dataclass-like repr that could be copy-pasted into a REPL. What I said was with regards to the current implementation.

Reply:
OK, but now I see that the above `repr` is not the _current_ `repr`, because VTK uses a custom `repr`.

Reply:
Thanks for your comment. I will leave this as is. Please comment if you notice something.